server/controllers/Untitled-1.js

```diff
@@ -0,0 +1,352 @@
+        valueLabel: 'Yes',
+        study: 'PI',
+        studyTarget: '5',
+        percent: 30,
+      },
+      {
+        color: '#97C0CE',
+        count: 2,
+        valueLabel: 'Yes',
+        study: 'LA',
+        studyTarget: '5',
+        percent: 20,
+      },
+      {
+        color: '#97C0CE',
+        count: 4,
+        valueLabel: 'Yes',
+        study: 'WU',
+        studyTarget: '5',
+        percent: 40,
+      },
+      {
+        color: '#97C0CE',
+        count: 2,
+        valueLabel: 'Yes',
+        study: 'PA',
+        studyTarget: '5',
+        percent: 20,
+      },
+      {
+        color: '#97C0CE',
+        count: 0,
+        valueLabel: 'Yes',
+        study: 'OR',
+        studyTarget: '5',
+        percent: 0,
+      },
+    ],
+    'N/A': [
+      {
+        color: 'grey',
+        count: 46,
+        valueLabel: 'N/A',
+        study: 'Totals',
+        studyTarget: '',
+        percent: 65.71428571428571,
+      },
+      {
+        color: 'grey',
+        count: 6,
+        valueLabel: 'N/A',
+        study: 'YA',
+        studyTarget: '',
+        percent: 60,
+      },
+      {
+        color: 'grey',
+        count: 4,
+        valueLabel: 'N/A',
+        study: 'NC',
+        studyTarget: '',
+        percent: 40,
+      },
+      {
+        color: 'grey',
+        count: 3,
+        valueLabel: 'N/A',
+        study: 'PI',
+        studyTarget: '',
+        percent: 30,
+      },
+      {
+        color: 'grey',
+        count: 4,
+        valueLabel: 'N/A',
+        study: 'LA',
+        studyTarget: '',
+        percent: 40,
+      },
+      {
+        color: 'grey',
+        count: 4,
+        valueLabel: 'N/A',
+        study: 'WU',
+        studyTarget: '',
+        percent: 40,
+      },
+      {
+        color: 'grey',
+        count: 8,
+        valueLabel: 'N/A',
+        study: 'PA',
+        studyTarget: '',
+        percent: 80,
+      },
+      {
+        color: 'grey',
+        count: 9,
+        valueLabel: 'N/A',
+        study: 'OR',
+        studyTarget: '',
+        percent: 90,
+      },
+    ],
+  },
+  title: 'Actigraphy consent (Mike)',
+  description:
+    'Displays % of participant who agreed to wear the activity watch',
+  legend: [
+    {
+      name: 'Yes',
+      symbol: {
+        type: 'square',
+        fill: '#97C0CE',
+      },
+    },
+    {
+      name: 'No',
+      symbol: {
+        type: 'square',
+        fill: '#60bc5a',
+      },
+    },
+  ],
+  studyTotals: {
+    Totals: {
+      count: 24,
+      targetTotal: 70,
+    },
+    YA: {
+      count: 4,
+      targetTotal: 10,
+    },
+    LA: {
+      count: 6,
+      targetTotal: 10,
+    },
+    WU: {
+      count: 6,
+      targetTotal: 10,
+    },
+    PA: {
+      count: 2,
+      targetTotal: 10,
+    },
+    PI: {
+      count: 3,
+      targetTotal: 10,
+    },
+    OR: {
+      count: 1,
+      targetTotal: 10,
+    },
+    NC: {
+      count: 2,
+      targetTotal: 10,
+    },
+  },
+}
```

Comment:
This file is labeled as Untitle-1.js..? does it belong to fixtures..?

Reply:
lol whoops - no this is not supposed to be checked in. I'll remove it!